reference/5-api-publisher/CloudWatch-configuration.md

```diff
@@ -0,0 +1,21 @@
+# Configuration CloudWatch
+
+Enables management of individual connection settings with encryption support for securely storing keys and secrets in the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html).
+
+## Configure AWS SDK
+
+Ensure that the AWS SDK has the necessary [configured AWS credentials](https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/net-dg-config-creds.html).
+
+## Configure API Publisher
+
+Configure the AWS log storage parameters in the configuration file.
+
+![](img/docker/logging-cloudwatch-configuration.png)
```

Comment:
All images need to have alt text with a brief description of the image. This could be "[Screenshot of Cloudwatch configuration]".

Reply:
fixed